ktor-features/ktor-gson/jvm/src/io/ktor/gson/GsonSupport.kt

```diff
@@ -40,7 +41,11 @@ class GsonConverter(private val gson: Gson = Gson()) : ContentConverter {
             throw ExcludedTypeGsonException(type)
         }
 
-        return gson.fromJson(reader, type.javaObjectType) ?: throw UnsupportedNullValuesException()
+        return try{
```

Comment:
Code formatting doesn't seem to be proper, missing spaces around try block.

Reply:
fixed